OpenRA.Mods.Common/Widgets/Logic/MapChooserLogic.cs

```diff
@@ -269,25 +317,24 @@ void EnumerateMaps(MapClassification tab, ScrollItemWidget template)
 					if (type != null)
 						details = type + " ";
 
-					details += $"({preview.PlayerCount} players)";
+					details += modData.Translation.GetString(Players, Translation.Arguments("players", preview.PlayerCount));
 					detailsWidget.GetText = () => details;
 				}
 
 				var authorWidget = item.GetOrNull<LabelWithTooltipWidget>("AUTHOR");
 				if (authorWidget != null)
-				{
-					WidgetUtils.TruncateLabelToTooltip(authorWidget, $"Created by {preview.Author}");
-				}
+					if (preview.Author != null)
+						WidgetUtils.TruncateLabelToTooltip(authorWidget, modData.Translation.GetString(CreatedBy, Translation.Arguments("author", preview.Author)));
```

Comment:
```
				if (authorWidget != null && preview.Author != null)
					WidgetUtils.TruncateLabelToTooltip(authorWidget, modData.Translation.GetString(CreatedBy, Translation.Arguments("author", preview.Author)));
```
This probably also wants to use `!string.IsNullOrEmpty(preview.Author)` instead of `preview.Author != null`.